lib/numbers/numbers.go

```diff
@@ -1,12 +1,6 @@
 package numbers
 
-type BetweenEqArgs struct {
-	Start  int
-	End    int
-	Number int
-}
-
 // BetweenEq - Looks something like this. start <= number <= end
-func BetweenEq(args BetweenEqArgs) bool {
-	return args.Number >= args.Start && args.Number <= args.End
+func BetweenEq(start, end, number int) bool {
```

Comment:
If you use `[T ~int | ~int32 | ~int64]` this will support `int`, `int32`, and `int64`. Could even throw `~float32 | ~float64` in there.
```
// BetweenEq - Looks something like this. start <= number <= end
func BetweenEq[T ~int | ~int32 | ~int64](start, end, number T) bool {
	return number >= start && number <= end
}
```

Reply:
Sure!